scripts/bulk-password-update-export.js

```diff
@@ -1,29 +1,33 @@
 
 process.env.NODE_TLS_REJECT_UNAUTHORIZED=0;
-const rpn = require('request-promise-native');
 const minimist = require('minimist');
 const {promises: fsPromises} = require('fs');
-const { randomInt } = require('crypto');
 const readline = require('readline');
+const rpn = require('request-promise-native');
+const {randomInt} = require('crypto');
+const csvSync = require('csv-parse/sync');
 
 const argv = minimist(process.argv.slice(2), {
   alias: {
     'h': 'help'
   }
 });
 
-if (argv.h|| !argv.url || !argv.user || !argv.password) {
-  console.log(`Change passwords of users in user-password-change.txt to random value, show results on screen 
```

Comment:
`user-password-change.txt` should be `user-password-change.csv`

Reply:
good call! fixed in 0bc2eba85